api/r/cellxgene.census/README.md

```diff
@@ -1,39 +1,79 @@
 
-# CZ CELLxGENE Discover Census
+# R package of CZ CELLxGENE Discover Census
 
 <!-- badges: start -->
 <!-- badges: end -->
 
+This is the documentation for the R package `cellxgene.census` which is part of Census by CZ CELLxGENE Discover. For full details on Census data and capabilities please go to the [main Census site](https://chanzuckerberg.github.io/cellxgene-census/).  
 
-The `cellxgene.census` package provides an API to facilitate the use of the CZ CELLxGENE Discover Census. For more information about the API and the project visit the [chanzuckerberg/cellxgene-census GitHub repo](https://github.com/chanzuckerberg/cellxgene-census/).
+`cellxgene.census` provides an API to efficiently access the cloud-hosted Census single-cell data from R. In just a few seconds users can access any slice of Census data using cell or gene filters across hundreds of single-cell datasets. 
 
-**Status**: Pre-release, under rapid development. Expect API changes.
-
-Also see the [Python API](https://cellxgene-census.readthedocs.io/).
+Census data can be fetched in an iterative fashion for bigger-than-memory slices of data, or quickly exported to basic R structures, as well as `Seurat` or `SingleCellExperiment` objects for downstream analysis.
 
 ## Installation
 
-You can install the development version of `cellxgene.census` from [GitHub](https://github.com/) with:
+From an R session, first install `tiledb` from R-Universe, the latest release in CRAN is not yet available.
```

Comment:
Since this is repeated in online docs, checking that you definitely want to repeat it here rather than linking out? It's not unreasonable, as is.